dist/obs-server.spec

```diff
@@ -308,7 +308,7 @@ rm src/api/Dockerfile.frontend-base
 # drop build script, we require the installed one from own package
 rm -rf src/backend/build
 
-find -name .keep | xargs rm -rf
+find -name .keep -o -name .gitignore | xargs rm -rf
```

Comment:
Why do we want to keep .gitignore now?

Reply:
and it is also not related with the commit

Reply:
This will remove them, not keep them. If we don't do that we get version-control-internal-file warnings